.gitignore

```diff
@@ -22,4 +39,4 @@ npm-debug.log
 yarn-debug.log
```

Comment:
yarn stuff ?

Reply:
again auto created but happy to remove if you want?